### DISCUSSION TITLE
Ayuda con Kata VUEJS

### DISCUSSION THREAD

**Discussion by Juan-2003** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#495] [Q&A]
¿Alguien le entendio como hacer la kata de VUEJS? Ya me vi todos los lives, pero no tengo la menor idea de como eso nos va a ayudar con enviar información a la página del pastelero, a la del cliente. etc

**Comment by Ingrzs** [comment]
sin duda alguna muchos estamos asi (me cuento), estoy haciendo esa parte XD
Mira el formulario los datos los vas a enviar y guardar que en este caso se guardaria en el estado.
Ya luego desde la pagina del pastelero mostraras los datos guardados que se enviarion del formulario y esos datos para mostrarlos usarias el getter creo...
Entenderlo no es dificil si no que no se sabe donde y en que parte va el codigo y si ademas el codigo lleva variables especificas ya que no solo es js tambien Vue

**Comment by ChristianDeM** [comment]
yo estaba en las mismasn  y llevo desde el  viernes estudiando desde la basico incluso mas de js.  si de plano no le entiendes bien te recomiendo aprender con el cdn  de manera local sin instalar nada.   lo que ando haciendo es crear 2 view el del front que es el del usuarios y otro el back que seria el pastelero.  cada  view esta echo de componentes  uno con la info otro  con lo de ingredientes etc...
para guardar los valores lo hago con v-model y methods con push.  ya tengo el primer view medio acabado.  a hora ando viendo como pasar la info del lado del pastelero. supondre que es con props  de echo sabia muy poco de js  desde el viernes ando estudiando todo eso por que igual ando en  blanco con vue.

**Reply by amendoxe** [reply]
Yo también me puse a estudiar desde lo básico =)

**Comment by martinGM05** [comment]
Em basicamente tienes que conocer que es un arreglo y que es un objeto, eso te servirá demasiado para almacenar esos datos en el estado.
